jb-manager-bot/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "jb-manager-bot"
```

Comment:
Rename the library to jb-manager-app

Reply:
In pypi the name of the library is jb-manager-bot. Checking if can be modifed.